src/main/java/uk/gov/hmcts/reform/sscs/personalisation/SyaAppealCreatedAndReceivedPersonalisation.java

```diff
@@ -270,8 +270,9 @@ private String convertBooleanToRequiredText(Boolean bool) {
         return bool ? REQUIRED : NOT_REQUIRED;
     }
 
-
     private String getOptionalField(String field, String text) {
```

Comment:
Please make this into a protected field and write a series of tests for it.  I am worried that in the past we have had a lot of null populated objects flying around and they have caused issues under different circumstances.  For example, `representative.getName().getFullNameNoTitle()` returns `null null` if `representative.getName().getFirstName()` is null and `representative.getName().getLastName()` is null.  Please write extensive tests for this method.

Reply:
I wonder if getFullName() (i.e. with Title) could give you "null null null". Wouldn't that be wild?
Perhaps I will change this to strip the string "null" and then test with StringUtils.isBlank()?

Reply:
Converted to Protected and added tests